awscli/topics/s3-config.rst

```diff
@@ -22,6 +22,7 @@ Configuration Values
 These are the configuration values you can set specifically for the ``aws s3``
 command set:
 
+* ``preferred_transfer_client`` - This determines the underlying Amazon S3 transfer client to use for transferring files to and from S3. For most workloads, choosing ``crt`` will enable faster uploads and downloads.
 * ``max_concurrent_requests`` - The maximum number of concurrent requests.
 * ``max_queue_size`` - The maximum number of tasks in the task queue.
```

Comment:
Some of these config options only apply to `classic`, and some only apply to `crt`. If we're going to suggest `crt` we probably shouldn't show a bunch of settings that don't affect it.

This is currently detailed further down in the doc: https://awscli.amazonaws.com/v2/documentation/api/latest/topic/s3-config.html#preferred-transfer-client

Should we do a better job in this doc of breaking up which options affect both, which affect crt-only, and which are classic-only?